src/3d/qgsvectorlayerchunkloader_p.cpp

```diff
@@ -168,15 +168,10 @@ QgsVectorLayerChunkLoaderFactory::QgsVectorLayerChunkLoaderFactory( const Qgs3DR
   , mSymbol( symbol->clone() )
   , mLeafLevel( leafLevel )
 {
-  QgsAABB rootBbox = Qgs3DUtils::mapToWorldExtent( context.extent(), zMin, zMax, context.origin() );
+  QgsBox3D rootBox3D( context.extent(), zMin, zMax );
```

Comment:
Same here, was this intentional?

Reply:
Yeah - we don't want world coordinates to be used in chunk nodes... world coordinates should be ideally only used for the setup of QTransforms (floats), and everywhere else we should be using map coordinates (doubles)